flycheck-ert.el

```diff
@@ -254,10 +254,7 @@ case, including assertions and setup code."
          (keys (car keys-and-body))
          (default-tags '(syntax-checker external-tool)))
     `(ert-deftest ,full-name ()
-       :expected-result
-       (list 'or
-             '(satisfies flycheck-ert-syntax-check-timed-out-p)
-             ,(or (plist-get keys :expected-result) :passed))
+       :expected-result ,(or (plist-get keys :expected-result) :passed)
```

Comment:
My understanding is that code was *passing* tests which timed out instead of failing them.  I ran into that for `rust-cargo` tests.  If they time out, I'd rather be notified, so I removed that bit.